content/rs/installing-upgrading/_index.md

```diff
@@ -72,6 +72,11 @@ After you download the .tar file installation package, install the package on on
 To install RS on Linux from the CLI:
 
 1. Copy the installation package to the node.
+
+    {{< note >}}
+For Ubuntu DEB packages and RHEL RPM packages, you can use the {{< download "GPG key file" "GPG-KEY-redislabs-packages.gpg" >}} to confirm authenticity of the package.
+    {{< /note >}}
```

Comment:
Can we add something in the spirit of:

To confirm authenticity on Ubuntu (DEB packages)
Import the public key by running
`gpg --import path/to/GPG-KEY-redislabs-packages
`
Then verify the package by running 
`dpkg-sig --verify <package file>
`

To confirm authenticity on RedHat (RPM packages)
Import the public key by running
`rpm --import path/to/redislabs-gpg-key
`
Then verify the package by running 
`rpm --checksig <package file>
`